src/components/sizePicker/sizePicker.tsx

```diff
@@ -0,0 +1,64 @@
+import { forwardRef, useState } from 'react';
+
+import { Text } from '../text';
+import classes from './sizePicker.module.scss';
+
+interface SizePickerProps {
+  defaultValue: string;
+}
+
+export const SizePicker = forwardRef<HTMLDivElement, SizePickerProps>(
+  function SizePicker({ defaultValue }, ref) {
+    const [checked, setChecked] = useState(defaultValue);
+
+    return (
+      <div ref={ref}>
+        <Text
+          variant="poppinsRegular"
+          color="lightGray"
+          className={classes.title}
+        >
+          Size
+        </Text>
+        <div className={classes.optionsCont}>
+          <label>
+            <input
+              type="radio"
+              name="size"
+              value="S"
+              onChange={(e) => setChecked(e.target.value)}
```

Comment:
Create a handler instead of arrow function:

```tsx
const handleSizeChange = (e: ChangeEvent<HTMLInputElement>) => {
  setChecked(e.target.value);
  onChacnge?.(e.target.value);
}

```